### ISSUE TITLE
[Issue]: Apple TV IDs shorted than 24 characters are not recognized

### ISSUE THREAD

**Issue by vevv** [MichaelYochpaz/iSubRip#46]
### Confirmations

- [X] I have checked the issues tab, and couldn't find an existing open issue for the issue I want to report.

### OS Type

Windows

### Python Version

3.10.6

### Package Version

2.4.2

### Description

Some Apple TV IDs are only 23 characters long.
Changing the regex here fixes the issue: https://github.com/MichaelYochpaz/iSubRip/blob/main/isubrip/scrapers/appletv_scraper.py#L20

Example: https://tv.apple.com/ca/movie/umc.cmc.sa30371iduf6frzmmat3mwh

### Output Log

```Text
Scraping https://tv.apple.com/ca/movie/umc.cmc.sa30371iduf6frzmmat3mwh
Error: No matching scraper was found for URL 'https://tv.apple.com/ca/movie/umc.cmc.sa30371iduf6frzmmat3mwh'
```

**Comment by MichaelYochpaz** [comment]
Hey, thanks for the report.
Noticed that as well. Seems to be that some of the new releases have shorter IDs.
Will be fixed in an update soon.

**Comment by MichaelYochpaz** [comment]
Fixed in the new release (2.4.3)
